Source/WebKit/UIProcess/API/APIContentWorld.cpp

```diff
@@ -88,13 +88,13 @@ Ref<ContentWorld> ContentWorld::sharedWorldWithName(const WTF::String& name, Opt
     return newContentWorld ? newContentWorld.releaseNonNull() : Ref { result.iterator->value.get() };
 }
 
-ContentWorld& ContentWorld::pageContentWorld()
+ContentWorld& ContentWorld::pageContentWorldSingleton()
 {
     static NeverDestroyed<RefPtr<ContentWorld>> world(adoptRef(new ContentWorld(WebKit::pageContentWorldIdentifier())));
```

Comment:
nit: Would be nice to use `Ref`

Reply:
You misunderstood. This function should not return a smart pointer. I'd like you to use `NeverDestroyed<Ref<ContentWorld>>`. Same in other places.

Reply:
Ah that makes sense. Fixed!